flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskFinalCheckpointsTest.java

```diff
@@ -483,130 +496,6 @@ public void acknowledgeCheckpoint(
         }
     }
 
-    @Test
-    public void testTriggerStopWithSavepointNoDrainWhenWaitingForFinalCheckpointOnSourceTask()
```

Comment:
test removal explanation: This has not been removed, but replaced/unified in `doTestTriggerStopWithSavepointWhenWaitingForFinalCheckpointOnSourceTask`